health-integrations/build.gradle

```diff
@@ -5,7 +5,9 @@ plugins {
 dependencies {
     compile project(':health-core')
     compile "com.netflix.eureka:eureka-client:${eurekaVersion}", optional
+    compile "com.netflix.archaius:archaius2-core:${archaiusVersion}"
```

Comment:
Adding archaius2 here means consumers of health-integrations will now be using a mix of archaius1 (from eureka) and archaius2. This is fine, but we should either (or maybe both):
1. add instructions on how to install the bridge module in the README
2. clean up eureka-client-archaius2 and use that here instead.